Database/Corrections/mopNPCFixes.lua

```diff
@@ -1370,6 +1370,10 @@ function MopNpcFixes.Load()
                 {44.41,16.8,phases.SKEER_IN_KLAXXI_VEES}
             }},
         },
+        [63123] = { -- Mogu Ritual Destroyed Credit
```

Comment:
The actual NPC IDs are 63087, 63088, 63089 and 63090. So I would use a killCreditObjective